esmvalcore/_config.py

```diff
@@ -172,13 +172,28 @@ def get_project_config(project):
 
 
 def get_institutes(variable):
-    """Return the institutes given the dataset name in CMIP5."""
+    """Return the institutes given the dataset name in CMIP5 and CMIP6."""
     dataset = variable['dataset']
     project = variable['project']
     logger.debug("Retrieving institutes for dataset %s", dataset)
+    try:
+        return CMOR_TABLES[project].institutes[dataset]
+    except (KeyError, AttributeError):
+        pass
     return CFG.get(project, {}).get('institutes', {}).get(dataset, [])
 
 
+def get_activity(variable):
+    """Return the activity given the experiment name in CMIP6."""
+    project = variable['project']
+    try:
+        exp = variable['exp']
```

Comment:
Note that `'exp'` can also be a list, e.g. `['historical', 'rcp85']` when using CMIP5 data. In that case the data from the two experiments will be concatenated.

Reply:
This is now addressed in the following lines.